libs/newt/Makefile

```
@@ -91,6 +91,8 @@ CONFIGURE_ARGS+= \
 
 MAKE_VARS+= PYTHON_CONFIG_PATH="$(STAGING_DIR)/host/bin"
 
+TARGET_LDFLAGS += $(if $(INTL_FULL),-lintl)
```

```suggestion
ifeq ($(CONFIG_BUILD_NLS),y)
  CONFIGURE_VARS+= ac_cv_lib_c_gettext=no
endif
```

`configure` does a test for `gettext` with `-lc` before trying `-lintl`, somehow it works and that's why `-lintl` is left out.

---

Probably because musl includes iconv and intl support. My previous tests were with uClibc-ng, which does not have iconv or intl support.

---

Actually, just had a thought. nls.mk specifies libintl-stub. So in theory this could always be passed.